Regression/Checksum/benchmarks_json/test_rz_laser_injection_from_lasy_file.json

```diff
@@ -1,13 +1,12 @@
 {
   "lev=0": {
```

Comment:
@dpgrote Same question.

Reply:
Because of the change in the boundary condition. This has been moved to PR https://github.com/BLAST-WarpX/warpx/pull/5886